crates/viewer/re_edit_ui/src/albedo_factor.rs

```diff
@@ -0,0 +1,21 @@
+use re_types::{components::AlbedoFactor, components::Color};
+use re_viewer_context::{MaybeMutRef, ViewerContext};
+
+use crate::color::edit_color_ui;
+
+pub fn edit_albedo_factor_ui(
+    ctx: &ViewerContext<'_>,
+    ui: &mut egui::Ui,
+    value: &mut MaybeMutRef<'_, AlbedoFactor>,
+) -> egui::Response {
+    let rgba = value.as_ref().0;
+
+    if let Some(value) = value.as_mut() {
+        let mut edit_color = Color(rgba);
+        let response = edit_color_ui(ctx, ui, &mut MaybeMutRef::MutRef(&mut edit_color));
+        value.0 = edit_color.0;
+        response
+    } else {
+        edit_color_ui(ctx, ui, &mut MaybeMutRef::Ref(&Color(rgba)))
+    }
```

Comment:
nit: This is fine as is, but we'll have this again, so we might as well add a `edit_rgba32`
```
pub fn edit_rgba32(
    _ctx: &re_viewer_context::ViewerContext<'_>,
    ui: &mut egui::Ui,
    value: &mut MaybeMutRef<'_, impl std::ops::DerefMut<Target = datatypes::Rgba32>>,
) -> egui::Response {
```
which forwards to non-generic `edit_rgba32_impl`.
The registration call in lib.rs then becomes `registry.add_singleline_edit_or_view::<AlbedoFactor>(edit_rgba32);`

I.e. the same pattern as with a bunch of the other more generic UIs